<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>colorPicker</class>
 <widget class="QWidget" name="colorPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>112</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="1">
    <widget class="mColorPicker" name="black">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="mColorPicker" name="red">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="mColorPicker" name="darkRed">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="mColorPicker" name="green">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="mColorPicker" name="darkGreen">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="mColorPicker" name="blue">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="mColorPicker" name="darkBlue">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="mColorPicker" name="cyan">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="mColorPicker" name="darkCyan">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="mColorPicker" name="magenta">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="mColorPicker" name="darkMagenta">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="mColorPicker" name="yellow">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="mColorPicker" name="darkYellow">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="mColorPicker" name="grey">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="mColorPicker" name="lightGrey">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="mColorPicker" name="white">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mColorPicker</class>
   <extends>QToolButton</extends>
   <header>mcolorpicker.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
